templates/kubernetes/docs/troubleshooting.md

```diff
@@ -115,6 +115,88 @@ Running the `juju-crashdump` script will generate a tarball of debug information
 
 ## Common Problems
 
+<a id="lxd"> </a>
```

Comment:
All titles get id's so you could remove this `id` and target `#charms-deployed-to-lxd-containers-fail-after-upgradereboot`